README.md

```diff
@@ -29,42 +29,27 @@ OpenTelemetry JS is under active development.
 This release isn't guaranteed to conform to a specific version of the specification, and future
 releases will not attempt to maintain backwards compatibility with the alpha release.
 
-The [v0.2 alpha
-release](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v0.2.0)
-includes:
-
-- Tracing API
-- Tracing SDK (Node and Web)
-- Metrics API
-- Jaeger Trace Exporter
-- Zipkin Trace Exporter
-- OpenTracing Bridge
-- HTTP, GRPC, DNS Instrumentations
-- Document Load for web
-- Metrics SDK (`Counter` and `Gauge` support) - Export work is underway.
-
-The [v0.3 alpha
-release](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v0.3.0)
-includes:
-
-- Prometheus Metric Exporter
-- OpenTelemetry Collector Exporter
-- mongodb, redis, mysql Instrumentations
+| Component                   | Version      | Release Date     |
+| --------------------------- | ------------ | ---------------- |
+| Tracing API                 | Alpha v0.1.0 | October 14 2019  |
+| Tracing SDK (Node and Web)  | Alpha v0.1.0 | October 14 2019  |
+| Jaeger Trace Exporter       | Alpha v0.1.0 | October 14 2019  |
+| Trace Context Propagation   | Alpha v0.1.0 | October 14 2019  |
+| Zipkin Trace Exporter       | Alpha v0.1.0 | October 14 2019  |
+| OpenTracing Bridge          | Alpha v0.1.0 | October 14 2019  |
+| Metrics API                 | Alpha v0.2.0 | November 04 2019 |
+| Metrics SDK                 | Alpha v0.2.0 | November 04 2019 |
+| Prometheus Metric Exporter  | Alpha v0.3.0 | December 13 2019 |
+| Collector Exporter          | Alpha v0.3.0 | December 13 2019 |
+| Resources                   | Beta         | March 25 2020    |
```

Comment:
Maybe `Beta v0.4.0`? Also it might be a good idea to link to the `Beta` epic?

Reply:
I was thinking about it but `v0.4.0` is already released.

Reply:
oh yea, I guess it's probably fine to keep a version out of it.

Reply:
shouldn't the beta be something like `1.0.0-beta-1`, `1.0.0-beta-2` and then finally we will have `1.0.0` ?

Reply:
probably a question for spec

Reply:
I defer to spec and check with other language maintainers.